x-pack/legacy/plugins/beats_management/wallaby.js

```diff
@@ -1,62 +0,0 @@
-/*
- * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
- * or more contributor license agreements. Licensed under the Elastic License;
- * you may not use this file except in compliance with the Elastic License.
- */
-const path = require('path');
-process.env.NODE_PATH = path.resolve(__dirname, '..', '..', '..', 'node_modules');
-
-module.exports = function (wallaby) {
-  return {
-    debug: true,
-    files: [
```

Comment:
We are not using `wallaby` anywhere else, and I'm not even sure this was working. Anyway this is not wired on any test suite or on the CI.